includes/express-checkout/class-wc-payments-express-checkout-button-helper.php

```diff
@@ -792,126 +792,6 @@ public function get_taxes_like_cart( $product, $price ) {
 		return WC_Tax::calc_tax( $price, $rates, false );
 	}
 
-	/**
```

Comment:
I moved these to `includes/express-checkout/class-wc-payments-express-checkout-ajax-handler.php`, since that's the only place they're used.